README.md

```diff
@@ -41,9 +43,19 @@ The command-line tool is called `jj` for now because it's easy to type and easy
 to replace (rare in English). The project is called "Jujutsu" because it matches
 "jj".
 
-If you have any questions, please join us on Discord
+Jujutsu is relatively young, with lots of work to still be done. If you have any
+questions, or want to talk about future plans, please join us on Discord
 [![Discord](https://img.shields.io/discord/968932220549103686.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/dkmfj3aGQN)
-or start a [GitHub Discussion](https://github.com/martinvonz/jj/discussions).
+or start a [GitHub Discussion](https://github.com/martinvonz/jj/discussions); the
+developers monitor both channels.
+
+> 🚧 **NOTE** 🚧: This is not a Google product. It is an experimental
```

Comment:
Optional: judging by @PhilipMetzger 's https://github.com/martinvonz/jj/pull/2158/files#diff-34e4f3b850d92bd27327004f3c62b0c52b5f6b7b83c61cdf7dc6583803836998R121, Github might recognize just `**Note**` or something like that and display it specially. If that's possible, I'd prefer that to the construction signs. Or, if you like the construction signs (they are pretty cool), you could put one in the beginning of the sentence (before "This").